runtime/mount/kernel.py

```diff
@@ -17,7 +17,9 @@
 from typing import TYPE_CHECKING, Any, Literal, TypedDict, TypeVar
 
 import numpy as np
+import orjson
```

Comment:
u want to replace `json` with `orjson` everywhere (probably primarily in `SocketIo`)